includes/class-ssp-frontend.php

```diff
@@ -442,12 +442,12 @@ public function get_album_art( $episode_id = false ) {
 		}
 
 		if ( $series_id ) {
-			$series_image = get_option( "ss_podcasting_data_image_{$series_id}", false );
+			$series_image = get_term_meta( $series_id, $this->token . '_series_image_settings', true );
 		}
 
 		if ( $series_image ) {
-			$series_image_attachment_id = ssp_get_image_id_from_url( $series_image );
-			$image_data_array           = $this->return_renamed_image_array_keys( wp_get_attachment_image_src( $series_image_attachment_id, 'medium' ) );
+			// A resized (medium) image might be cropped, we need to use the original image here.
```

Comment:
If I recall correctly, the reason we're doing this is that the image needs to be perfectly square to submit the feed to iTunes. If the user uploads a portrait or landscape image, using the resized image will ensure that it is cropped to be square.